### PR TITLE
fix: Preselect Ticket holder same as Ticket Buyer in Order Form

### LINKED ISSUE #5851: Preselect "Ticket holder" same as "Ticket Buyer
Always preselect "Ticket holder is the same as ticket buyer." by default, but keep functionality as is. So, the ticket buyer can untick the box if they want.

Screenshot from 2020-12-01 20-43-03

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes #5851 

#### Short description of what this resolves:
Checkbox for selecting ticket holder same as ticket buyer is checked initially (with details displayed). User can toggle to turn it off.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by MeghalBisht** [comment]
Just noticed that if someone selects more than one ticket all the tickets will get the same name as the buyer. What can be a better solution for this?

**Comment by mariobehling** [comment]
> Just noticed that if someone selects more than one ticket all the tickets will get the same name as the buyer. What can be a better solution for this?

This only happens in your PR, not in the current system. Only the first person should have the name added even if the ticket holder field is pre-selected.

File: app/components/forms/orders/order-form.js [hunk 1]
```diff
@@ -25,9 +25,9 @@ export default Component.extend(FormMixin, {
   buyerHasLastName  : readOnly('data.user.lastName'),
   holders           : computed('data.attendees', function() {
     this.data.attendees.forEach(attendee => {
-      attendee.set('firstname', '');
-      attendee.set('lastname', '');
-      attendee.set('email', '');
+      attendee.set('firstname', this.buyerFirstName);
```
Comment (maze-runnar) [inline app/components/forms/orders/order-form.js:28]: set it only for first attendee.

Comment (MeghalBisht) [reply app/components/forms/orders/order-form.js:28]: Understood, thanks!

**Comment by MeghalBisht** [comment]
> > Just noticed that if someone selects more than one ticket all the tickets will get the same name as the buyer. What can be a better solution for this?
> 
> This only happens in your PR, not in the current system. Only the first person should have the name added even if the ticket holder field is pre-selected.

Yes, I noticed the bug in my PR after I opened it. Hope it's good now

**Review by maze-runnar — APPROVED** [review]

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**